app/src/main/res/layout/selection_interaction_item.xml

```diff
@@ -19,24 +23,37 @@
     android:descendantFocusability="beforeDescendants"
     android:focusableInTouchMode="true"
     android:orientation="vertical"
-    android:padding="0dp"
+    android:paddingTop="12dp"
+    android:paddingBottom="12dp"
     app:layoutMarginEnd="@{viewModel.hasConversationView ? @dimen/margin_24 : @dimen/margin_32}"
     app:layoutMarginStart="@{viewModel.hasConversationView ? @dimen/margin_40 : @dimen/margin_32}"
     app:layout_constraintBottom_toBottomOf="parent"
     app:layout_constraintTop_toTopOf="parent">
 
+    <TextView
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="16dp"
+      android:layout_marginTop="4dp"
+      android:layout_marginEnd="16dp"
+      android:layout_marginBottom="4dp"
+      android:fontFamily="sans-serif-light"
+      android:text="@string/item_selection_text"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="14sp"
+      android:textStyle="italic"
+      android:visibility="@{viewModel.getSelectionItemInputType() == SelectionItemInputType.RADIO_BUTTONS ? View.GONE : View. VISIBLE}" />
+
     <org.oppia.app.player.state.SelectionInteractionView
       android:id="@+id/selection_interaction_recyclerview"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:layout_marginTop="@dimen/divider_margin_top"
-      android:layout_marginBottom="@dimen/divider_margin_bottom"
+      android:layout_marginStart="12dp"
+      android:layout_marginEnd="12dp"
```

Comment:
**Comment 7**

Keeping it margin `12dp` instead of margin `16dp` because as mentioned in description the `Radio/Checkbox` already contains `4dp` padding by default.